VirtoCommerce.Storefront/Controllers/Api/ApiPricingController.cs

```diff
@@ -40,5 +48,44 @@ public async Task<ActionResult<ProductPrice[]>> GetActualProductPrices([FromBody
             }
             return Ok();
         }
+
+        // POST: storefrontapi/pricing/total
+        [HttpPost("total")]
+        public async Task<ActionResult<ProductPrice>> GetProductsTotal([FromBody] AddCartItem[] items)
+        {
+            // Based on https://github.com/VirtoCommerce/vc-module-cart/blob/05dac9453f375f015bbea57e3f95f4c87a62ec74/src/VirtoCommerce.CartModule.Data/Services/DefaultShoppingCartTotalsCalculator.cs#L57-L68
+            if (items != null && items.Any())
```

Comment:
Can we invert if statement to reduce cognitive complexity?

Reply:
Our convention is to use if-else blocks instead. Also I didn't implemented another proposal - to have single return - because returning `Ok` without parameters and `Ok` with null are not the same.